jgclark.NoteHelpers/README.md

```diff
@@ -27,6 +27,7 @@ This plugin provides commands to do things with notes that aren't yet provided i
 - **list inconsistent note filenames**: lists the names of notes whose filenames are inconsistent with their titles
 - **rename filename to title**: renames the current filename to the title of the note
 - **rename inconsistent note filenames**: renames the files of notes whose filenames are inconsistent with their titles. When run it offers to rename all in one go, or to go one by one so you can choose which ones to rename.
+- **Find unlinked notes**: Finds and creates links to existing notes in the current note.
```

Comment:
Add "(It does this by looking for exact matches to all existing note titles. It ignores text in code blocks, in markdown links, and surrounded by punctuation. The command may take a little time to run.)"